routers/web/repo/compare.go

```diff
@@ -800,6 +802,47 @@ func CompareDiff(ctx *context.Context) {
 			if ctx.Written() {
 				return
 			}
+
+			// Get reviewer info for pr
```

Comment:
Maybe this can be moved into RetrieveRepoMetas, where similar code exists for populating `ctx.Data["Assignees"]`?